cylc/flow/network/schema.py

```diff
@@ -1466,7 +1466,7 @@ def description(self):
 class Broadcast(Mutation):
     class Meta:
         description = sstrip('''
-            Override or add new [runtime] config in targeted namespaces.
+            Override or add new [runtime] configurations in a running workflow.
```

Comment:
Note: these descriptions can contain markdown.
```suggestion
            Override or add new `[runtime]` configurations in a running workflow.
```